src/blueapi/service/main.py

```diff
@@ -420,6 +421,12 @@ def set_state(
     return runner.run(interface.get_worker_state)
 
 
+@router.get("/scratch", response_model=ScratchResponse)
+@start_as_current_span(TRACER)
+def get_scratch_packages(runner: WorkerDispatcher = Depends(_runner)):
+    return runner.run(interface.get_scratch)
```

Comment:
Must: I don't think we need to interrogate the scratch area from inside the subprocess, if you do the logic you've put in `interface.py` here instead would it change anything?

Reply:
@ZohebShaikh you mentioned this to me and thinking about it: nevermind. The subprocess' job is to make managing the scratch area easier so the logic probably belongs in there, can leave this as-is.